src/qt/pivx/settings/settingswalletoptionswidget.cpp

```diff
@@ -130,6 +127,11 @@ void SettingsWalletOptionsWidget::discardWalletOnlyOptions()
     reloadWalletOptions();
 }
 
+void SettingsWalletOptionsWidget::saveMapPortOptions()
+{
+    ClientModel::mapPort(ui->mapPortUpnp->isChecked(), ui->mapPortNatpmp->isChecked());
```

Comment:
Following the previous comment that left, here you can access `clientModel` directly instead of calling to the static function.

Reply:
done